src/components/ui/carousel.tsx

```
@@ -1,262 +1,212 @@
+                {...props}
+            >
+                <ChevronLeft className="h-4 w-4 !stroke-amber-50" />
+                {/* <span className="sr-only">Previous slide</span> */}
+            </Button>
+        );
+    }
+);
+CarouselPrevious.displayName = "CarouselPrevious";
+
+const CarouselNext = React.forwardRef<HTMLButtonElement, React.ComponentProps<typeof Button>>(
+    ({ className, variant = "outline", size = "icon", ...props }, ref) => {
```

ESLint

[error] 'variant' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)